spec/draft-barnes-cfrg-4q.md

```diff
@@ -491,18 +503,21 @@ return Q
 
 ~~~~~
 
-To negate an R2 point (N,D, E, F) one computes (D, N, E ,-F). It is
-important to do this and the table lookup in constant time.  This
-multiplication algorithm only works properly for N-torsion
+Negation of a point can be required after fetching an R2 point from the table T. To negate an R2 point (N, D, E, F) 
+one computes (D, N, E ,-F). It is important to do this (as well as the table lookup) in constant time.  
+
+The multiplication algorithm above only works properly for N-torsion
 points. Implementations for Diffie-Hellman key exchange (and similar
 applications) MUST NOT use this algorithm on anything that is not a
 torsion point.  Otherwise, it will produce the wrong answer, with
 consequences for security.
 
-# Use of the scalar multiplication primitive in Diffie-Hellman
+# Use of the scalar multiplication primitive for Diffie-Hellman Key Agreement
 
 The above scalar multiplication primitive can be used to implement elliptic curve Diffie-Hellman
-with cofactor. The multiplication by the cofactor 392 requires nine doublings and two additions.
+with cofactor.  Note that the multiplication by the cofactor 392 does not need to be computed in
+constant time and, hence, it only requires nine doublings and two additions (i.e., one uses the binary 
```

Comment:
I think I was very confused this morning. Yes, you don't need constant time and 392 is known so you can use a short addition chain. For DH points are usually public. This doesn't make the routine variable time though: that arithmetic is still constant time.